packages/server/src/lib/userImporter.js

```diff
@@ -70,13 +76,19 @@ class UserImporter {
                 && (existingUser.tenant_id === adminUser.tenant_id)) {
                 return NOT_CHANGED;
             }
+            // Shouldn't get here until we enable the update functionality. See existing user check in checkRow().
             await UserImporter.syncUser(newUser);
             return UPDATED;
         }
         await db.createUser(newUser);
+        // Don't send emails when running automated tests.
```

Comment:
Once this comment is implemented, you shouldn't need this additional conditional and argument to be passed in: https://github.com/usdigitalresponse/usdr-gost/pull/768/files#r1062787890